e2e/cypress/tests/integration/enterprise/overage_users/error_overage_users_spec.ts

```diff
@@ -27,6 +29,15 @@ const withEmailLicense = (email = emailTest) => {
     });
 };
 
+const withSubscriptionExpendable = (isExpendable: boolean) => {
```

Comment:
It's expendable, no expensable. It's a typo as `d` and `s` are close.

Reply:
Following up here. I agree its not `expensable`. I think it should be an `a` instead of an `e`, `expandable`. `expendable` means a one time thing that we don't worry about if we discard it after we use it. `expandable` means we can make it larger. If I understand this and the accompanying server change, we want `expandable` everywhere that currently says `expendable`.